.gitignore

```diff
@@ -1,4 +1,5 @@
 /node_modules/
 /public/build/
+/venv
```

Comment:
Note that putting the forward slash before the folder makes this relative to the root `venv` subdirectories won't be caught.

Reply:
Thanks! I didn't take these into consideration. Creating a commit with suggested changes.